### PR TITLE
feat(EntityList): enable delete many

### PR DESCRIPTION
**Pull request by haayhappen**

### CODE REVIEW

File: src/entities/_base/index.ts [hunk 1]
```diff
@@ -413,54 +413,54 @@ export abstract class UniverseEntity<Payload, RawPayload> extends Entity<Payload
...
-  constructor (public message: string = 'Could not alter resource unexpectedly.', properties?: any) {
+  constructor(public message: string = 'Could not alter resource unexpectedly.', properties?: any) {
     super(message, properties)
   }
 }
 
 export class EntityFetchError extends BaseError {
   public name = 'EntityFetchError'
-  constructor (public message: string = 'Could fetch resource unexpectedly.', properties?: any) {
+  constructor(public message: string = 'Could fetch resource unexpectedly.', properties?: any) {
```
Comment (eljefedelrodeodeljefe) [inline src/entities/_base/index.ts:451]: there is some misconfig on the linter

**Review by eljefedelrodeodeljefe — COMMENTED** [review]
Generally LGTM minus the linter

File: src/entities/_base/index.ts [hunk 2]
```diff
@@ -633,4 +641,48 @@ export abstract class EntitiesList<Entity, RawPayload> extends Readable {
...
+   */
+  public async delete (payload: IEntitiesListDeleteManyPayload, options?: IEntitiesListDeleteManyOptions): Promise<string[] | [] | undefined> {
+    // we allow implementers to override us by calling ._fetch directly and e.g. handle our error differently
+    return await this._delete(payload, options)
+  }
+
+  /**
+   * @ignore
+   */
+  protected async _delete (payload: IEntitiesListDeleteManyPayload, options?: IEntitiesListDeleteManyOptions): Promise<string[] | [] | undefined> {
```
Comment (artdnlv) [inline src/entities/_base/index.ts:656]: is that `[]` really necessary?

File: src/entities/notification-campaign/notification-campaign.ts [hunk 3]
```diff
@@ -581,9 +581,43 @@ export class NotificationCampaign extends UniverseEntity<NotificationCampaignPay
...
+  protected parseItem(payload: NotificationCampaignRawPayload): NotificationCampaign {
+    return NotificationCampaign.create(payload, this.universe, this.http)
+  }
+
+  public async getStream(options?: UniverseFetchOptions): Promise<NotificationCampaigns> {
+    return (await this._getStream(options)) as NotificationCampaigns
+  }
+
+  public async exportCsv(options?: UniverseExportCsvOptions): Promise<Blob> {
+    return (await this._exportCsv(options))
```
Comment (artdnlv) [inline src/entities/notification-campaign/notification-campaign.ts:611]: not sure if this is in line with overall concept but you don't need to await here, you can just `return this._exportCsv(...)`. Same is fair for other cases.

File: src/entities/_base/index.ts [hunk 4]
```diff
@@ -531,7 +539,7 @@ export abstract class EntitiesList<Entity, RawPayload> extends Readable {
       this.push(this.parseItem(result.value))
 
       reader.read()
-      // @ts-expect-error
+        // @ts-expect-error
```
Comment (artdnlv) [inline src/entities/_base/index.ts:542]: 😭

**Review by artdnlv — APPROVED** [review]

**Pull request merged**